images/capi/hack/image-build-ova.py

```diff
@@ -118,6 +118,8 @@ def main():
                  "centos8-64": {"id": "107", "version": "8", "type": "centos8-64"},
                  "rhel7-64": {"id": "80", "version": "7", "type": "rhel7_64guest"},
                  "ubuntu-64": {"id": "94", "version": "", "type": "ubuntu64Guest"},
+                 "coreos-64": {"id": "101", "version": "", "type": "linux-64"},
```

Comment:
Need to double-check the IDs. Couldn't find any reference docs for them.

Reply:
I should have put more info in the commit message. A quick search gives https://github.com/josenk/vagrant-vmware-esxi/blob/master/ESXi_guestos_types.md

Reply:
also see https://github.com/kinvolk/image-builder/pull/11/commits/679fbfc8e855d4fffc846e4d7151f93841cf8d8c